tests/app/flask_search_api_client/test_search_api_client.py

```diff
@@ -0,0 +1,127 @@
+from app.flask_search_api_client.search_api_client import SearchApiClient
+from nose.tools import assert_equal
+import requests
+import requests_mock
+import os
+from flask import json
+from app import create_app
+
+
+class TestSearchApiClient():
+
+    search_api_client = None
+
+    def __init__(self):
+        os.environ['ES_ENABLED'] = 'True'
```

Comment:
How does this work? It's setting an environment value, but it doesn't look like it's accessed anywhere. Is this relying on the flask env to config? Can we set/patch the config directly instead?